### ISSUE TITLE
Add. 14,593, foll. 1–185 | https://bl.syriac.uk/ms/329

### ISSUE THREAD

**Issue by DA-Gharib** [srophe/britishLibrary-data#1866]
The ms in the pdf does not specify whether the book of the "Κλῖμαξ" by John Climacus is just an index of the chapters as described in the XML. 
Compare https://archive.org/details/catalogueofsyria02brituoft/page/590/mode/1up?view=theater with the second <note type="contents-note"> in the XML.

**Comment by wlpotter** [comment]
Thanks @DA-Gharib! It looks like Wright tells us at the bottom of the right-hand column of page 590 that there is an index to the Κλῖμαξ on fol. 1 b that was added by a later hand. I think Wright included it at the end because it is a more modern addition. However, we have tried to include indexes, tables of content, fihrists, and so forth as manuscript items as much as possible, which is why the encoded added the index as the first msItem. The note that says "See additions" refers to addition 8 which says that it is a later hand's index.

In some cases we would move the index as a child of the msItem for the Κλῖμαξ, but in this case, since it is a later hand that added it, I believe it makes sense to keep it as a separate a-level item, as it is currently.
